### DISCUSSION TITLE
Resistor Help

### DISCUSSION THREAD

**Discussion by TonyM4119** [nebhead/PiFire#114] [Q&A]
I need some help with buying the resistors, I know the sizes but the first ones I ordered were 10k ohm 1 watt, 1k ohm 1 watt and 330 ohm 1 watt. The problem is that the physical size of them was huge and didn't fit the pifire board. Can someone please tell me what watt size I need.

**Comment by nebhead** [comment]
1/4 Watt should work just great.  I would say you could even go up to 1/2 watt, but I believe that the board is designed with 1/4 Watt resistors in mind.  1 Watt is definitely going to be too large.  I suppose you could sort of mount them vertically if needed.  Doesn't have to be too pretty.

**Comment by TonyM4119** [comment]
Awesome thanks so much for getting back to me so quickly. I emailed you all
the other day because I'm planning on attempting to get the PiFire to work
on the new Traeger models if it's something you're interested in helping me
with that'd be awesome.

**Reply by nebhead** [reply]
No problem!  I just found your other e-mail in my junk mail folder (thanks Google), and responded.  I think some of the folks on here would definitely be interested in getting the new Traegers working with PiFire.  I think we were speculating it probably works off of a 12V or 24V supply, and would require another power supply for those voltages.  Keep us posted and feel free to ask questions in the discussions groups.
